Include/internal/pycore_typeobject.h

```diff
@@ -49,8 +49,6 @@ typedef struct {
     // XXX tp_dict, tp_bases, and tp_mro can probably be statically
```

Comment:
should this comment (added by the original PR) be updated now that bases and mro don't exist here?

Reply:
fixed